packages/fuels-test-helpers/src/lib.rs

```diff
@@ -127,13 +128,24 @@ pub fn setup_single_asset_coins(
     coins
 }
 
+pub fn setup_single_message(owner: &Bech32Address, amount_per_message: u64) -> Vec<Message> {
+    let message = Message {
+        owner: owner.into(),
+        amount: amount_per_message,
+        ..Default::default()
+    };
+
+    vec![message]
+}
+
 // Setup a test client with the given coins. We return the SocketAddr so the launched node
 // client can be connected to more easily (even though it is often ignored).
 #[cfg(feature = "fuel-core-lib")]
 pub async fn setup_test_client(
     coins: Vec<(UtxoId, Coin)>,
     node_config: Option<Config>,
     consensus_parameters_config: Option<ConsensusParameters>,
+    messages: Option<Vec<Message>>,
```

Comment:
This might be picky, but can we move this param to be right after `coins`? I feel like the function makes more sense if they're next to each other.